blobs/xx30/vbios_t530.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+BLOBDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROM_PARSER_SHA256SUM="082a04e789d2343a01649327a7c9ba5ae5186a2b38e3d9de15c078845e732819  master.zip"
+UEFI_EXTRACT_SHA256SUM="c9cf4066327bdf6976b0bd71f03c9e049ae39ed19ea3b3592bae3da8615d26d7  UEFIExtract_NE_A58_linux_x86_64.zip"
+VBIOS_FINDER_SHA256SUM="bbffe3c9c8a64c31485618fae2a2892fe2f4da7b45adaaba136fd45056bce9cd  master.zip"
+BIOS_UPDATE_SHA256SUM="6fc652b5fa10e5ea1ddc0e8477a2f1cfe56e00df76a94629f9b736faaee6199c  g4uj41us.exe"
+DGPU_ROM_SHA256SUM="3efe4886530086c0b612d1e669fbb4459ec93ec949b25a909e7ad273f4f01015  vbios_10de_0def_1.rom"
+IGPU_ROM_SHA256SUM="10b292c19322e7bb7db53350d2775d37b72a784292ea5686cd0f92af929f4916  vbios_8086_0106_1.rom"
+echo "### Creating temp dir"
+extractdir=$(mktemp -d)
+cd "$extractdir"
+
+echo "### Installing basic dependencies"
+sudo apt update
+sudo apt install -y wget ruby ruby-dev ruby-bundler p7zip-full upx-ucl 
+
+echo "### Downloading rom-parser dependency"
+wget https://github.com/awilliam/rom-parser/archive/refs/heads/master.zip
+
+echo "### Verifying expected hash of rom-parser"
+echo "$ROM_PARSER_SHA256SUM" | sha256sum --check || { echo "Failed sha256sum verification..." && exit 1; }
+
+echo "### Installing rom-parser dependency"
+unzip master.zip
+rm master.zip
+cd rom-parser-master
+make
+sudo cp rom-parser /usr/sbin/
+cd ..
+rm -r rom-parser-master
+
+echo "### Downloading UEFIExtract dependency"
+wget https://github.com/LongSoft/UEFITool/releases/download/A58/UEFIExtract_NE_A58_linux_x86_64.zip
+
+echo "### Verifying expected hash of UEFIExtract"
+echo "$UEFI_EXTRACT_SHA256SUM" | sha256sum --check || { echo "Failed sha256sum verification..." && exit 1; }
+
+echo "### Installing UEFIExtract"
+unzip UEFIExtract_NE_A58_linux_x86_64.zip
+sudo mv UEFIExtract /usr/sbin/
+rm UEFIExtract_NE_A58_linux_x86_64.zip
+
+echo "### Downloading VBiosFinder"
+wget https://github.com/coderobe/VBiosFinder/archive/refs/heads/master.zip
```

Comment:
Same here. We do not want to point to master.
Clicking on lastest commit https://github.com/coderobe/VBiosFinder/commit/c2d764975115de466fdb4963d7773b5bc8468a06 then browse file and then "Code" button shows the URL https://github.com/coderobe/VBiosFinder/archive/c2d764975115de466fdb4963d7773b5bc8468a06.zip under download button. This is the archive, for that commit ID, that you can force a sha256sum verification hash inside of the code that will work forever. That is, whoever will download that archive will download tree until that commit. This should be measurable and the same in the future. If this is not, then Huston we have a problem.